lib/tezos/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "aws-blockchain-node-runners-tz",
+  "version": "0.1.0",
+  "bin": {
+    "scroll": "bin/tz.js"
+  },
+  "scripts": {
+    "build": "tsc",
+    "watch": "tsc -w",
+    "test": "jest",
+    "cdk": "cdk",
+    "cdk_deploy_common": "cdk deploy tz-common",
+    "cdk_deploy_ha_nodes": "cdk deploy tz-ha-nodes",
+    "cdk_destroy_common": "cdk destroy tz-common",
+    "cdk_destroy_ha_nodes": "cdk destroy tz-ha-nodes"
+  },
+  "devDependencies": {
```

Comment:
Please remove all dependencies. We switched to using only the dependencies from the root directory of the node-runners repository.